api/Node.json

```diff
@@ -397,7 +397,7 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "≤4"
```

Comment:
https://trac.webkit.org/browser/webkit/trunk?rev=4552 is where Safari 1 branched from WebKit trunk, so searching the repo at that time can help determine if something was in Safari 1. Unfortunately `compareDocumentPosition` wasn't in the tree then, so I can't suggest an improvement here :)